cdap-proto/src/main/java/co/cask/cdap/proto/metadata/lineage/FieldOperationInfo.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.proto.metadata.lineage;
+
+/**
+ * Represents field lineage operation.
+ */
+public class FieldOperationInfo {
```

Comment:
this needs better documentation of what the fields mean. What is the id, what is the name, what is the difference between them?

Reply:
Updated the javadoc. However seems like we don't need the `id` field in the operation. We can ask app to provide the unique name for it. The purpose of `id` field was in plugins , user can specify the the simple operation name say `normalize` and then platform generates unique id for it, however app will anyway have to prefix it with stage-names, pipeline-names etc to uniquely identify operations across different stages.